Compile error under Visual Studio 2015 when wrong order of #include directives

The code

```
#include <spdlog/sinks/stdout_sinks.h>
#include <spdlog/spdlog.h>

int main() {
	spdlog::set_level(spdlog::level::debug);
	auto logger = spdlog::stdout_logger_st("stdout");
	spdlog::get("stdout")->debug("Log entry");
}
```

does not compile under Microsoft Visual Studio Community 2015 (Version 14.0.25431.01 Update 3). Switching the #include's removes the compile error.

@katastrofa999 Please remember to include the actual error message next time :)

So the error presents as follows:
```
1>c:\projects\spdlog\include\spdlog\details\console_globals.h(20): error C3646: 'handle': unknown override specifier
1>c:\projects\spdlog\include\spdlog\details\console_globals.h(20): error C2059: syntax error: ')'
1>c:\projects\spdlog\include\spdlog\details\console_globals.h(21): error C2334: unexpected token(s) preceding '{'; skipping apparent function body
```

It seems that include to "os.h" is missing in console_globals.h. I'll prepare a PR.

--- a/include/spdlog/details/console_globals.h
+++ b/include/spdlog/details/console_globals.h
@@ -9,7 +9,16 @@
 #include <mutex>
 
 #ifdef _WIN32
-#include "spdlog/details/os.h"
+
+#ifndef NOMINMAX
+#define NOMINMAX // prevent windows redefining min/max
+#endif
+
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#include <windows.h>
 #endif
 
 namespace spdlog {

--- a/include/spdlog/details/console_globals.h
+++ b/include/spdlog/details/console_globals.h
@@ -8,6 +8,10 @@
 #include <cstdio>
 #include <mutex>
 
+#ifdef _WIN32
+#include "spdlog/details/os.h"
+#endif
+
 namespace spdlog {
 namespace details {
 struct console_stdout

